src/sources/logplex.rs

```diff
@@ -65,7 +69,7 @@ impl HttpSource for LogplexSource {
 }
 
 #[async_trait::async_trait]
-#[typetag::serde(name = "logplex")]
+#[typetag::serde(name = "heroku_logs")]
```

Comment:
I see you have added a compatibility source description through `inventory::submit` above. I think we also need a compatibility alias here as well.

Reply:
True, shame that there isn't some neat alias attribute for this.